app/extensions/elasticsearch/__init__.py

```diff
@@ -1312,7 +1312,7 @@ def es_status(app=None, outdated=True, missing=False, active=True, health=True):
             status['elasticsearch:enabled'] = False
 
         health_data, stats_data = es_health_and_stats(app=app)
-        if health_data['status'] != 'green':
+        if health_data['status'] not in ('green', 'yellow'):
```

Comment:
I want to keep the yellow status check, but instead make the tests ignore a yellow status.  See here for potential fix: 0a9a867de6f31ba8498b0d28b3d300b77aea8e5d

Reply:
This fix has been merged into main, so this health status check can be deleted.

Reply:
Thanks!  I've removed the elasticsearch commits